evalutils/templates/algorithm/cookiecutter.json

```diff
@@ -1,5 +1,10 @@
 {
     "algorithm_name": "",
+    "algorithm_kind": [
+        "Classification",
+        "Segmentation",
+        "Detection"
```

Comment:
Do people have to type this out? Would be nice if they could use `C`, `S` or `D`. Maybe another issue?

Reply:
Another issue might be better, this is in line with the Evaluations. I'll make one!

Reply:
#291